test/unit/range/views/view_kmer_hash_test.cpp

```diff
@@ -17,12 +19,31 @@
 #include <seqan3/range/views/take_until.hpp>
 #include <seqan3/range/views/to.hpp>
 
-#include <gtest/gtest.h>
+#include "../iterator_test_template.hpp"
 
 using seqan3::operator""_dna4;
 using seqan3::operator""_dna5;
 using seqan3::operator""_shape;
 
+using iterator_type = std::ranges::iterator_t<decltype(std::declval<seqan3::dna4_vector&>()
+                                                        | seqan3::views::kmer_hash(0b1001_shape))>;
+
+template <>
+struct iterator_fixture<iterator_type> : public ::testing::Test
+{
+    using iterator_tag = std::forward_iterator_tag;
+    static constexpr bool const_iterable = true;
+
+    seqan3::dna4_vector text{"ACGTAGC"_dna4};
+
+    decltype(seqan3::views::kmer_hash(text, 0b101_shape)) test_range = seqan3::views::kmer_hash(text, 0b101_shape);
```

Comment:
you are aware that here the shape is `0b101` and in the iterator defintion `0b1001`?

Reply:
nope. Copy and paste error. It will not affect the type but it should be consistent anyway